app/controllers/api/v2/containers_controller.rb

```diff
@@ -38,13 +38,16 @@ def show
           param :name, String, :required => true
           param_group :taxonomies, ::Api::V2::BaseController
           param :compute_resource_id, :identifier, :required => true
-          param :registry_id, :identifier, :desc => N_('Registry this container will have to use
-                                                        to get the image')
-          param :image, String, :desc => N_('Image to use to create the container.
-                                            Format should be repository:tag, e.g: centos:7')
+          param :registry_id, :identifier, :desc => N_('Registry this container will have to
+                                                        use to get the image')
+          param :repository_name, String, :required => true,
+                                          :desc => N_('Name of the repository to use
+                                                      to create the container. e.g: centos')
+          param :tag, String, :required => true,
```

Comment:
I'm wondering if we should not require this and just use "latest" if they pass no tag?